user/src/main/java/com/hufudb/openhufu/user/OpenHuFuUser.java

```diff
@@ -97,6 +98,19 @@ public static void main(String[] args) {
     }
   }
 
+  public ResultSet executeTask(WXY_UserConfig userConfig) throws SQLException {
+    for (String endpoint: userConfig.endpoints) {
+      addOwner(endpoint);
+    }
+    for (GlobalTableConfig config: userConfig.globalTableConfigs) {
+      createOpenHuFuTable(config);
+    }
+    LOG.info("Init finish");
+    Statement statement = createStatement();
+    ResultSet resultSet = statement.executeQuery(userConfig.sql);
+    return resultSet;
```

Comment:
Statement  object 'statement' will be leaked, because of missing of 'statement.close()'.
This is a common problem in OpenHuFU